### PR TITLE
Add UI bug template

### PR DESCRIPTION
**Pull request by harupy**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

Add a UI bug template to encourage people to do the following:

- Record steps to reproduce the issue as a video or GIF, which is much easier to understand than text and more reproducible.
- Check the console and network panels in the developer tools to get more information about the issue.

## How is this patch tested?

Manually tested on my fork:

https://github.com/harupy/mlflow/issues/new?assignees=&labels=bug%2Carea%2Fuiux&template=ui_bug_report_template.yaml&title=%5BBUG%5D

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly by following the steps below.

1. Check the status of the `ci/circleci: build_doc` check. If it's successful, proceed to the
   next step, otherwise fix it.
2. Click `Details` on the right to open the job page of CircleCI.
3. Click the `Artifacts` tab.
4. Click `docs/build/html/index.html`.
5. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [x] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [inline .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:73]:
```suggestion
        ![console-panel](https://github.com/mlflow/mlflow/blob/master/.github/ISSUE_TEMPLATE/images/console-panel.png?raw=true)
```

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [inline .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:81]:
```suggestion
        ![network-panel](https://github.com/mlflow/mlflow/blob/master/.github/ISSUE_TEMPLATE/images/network-panel.png?raw=true)
```

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (BenWilson2) [inline .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:69]: Perhaps a README that details how to get to a relevant UI issue in Chrome and Firefox would be helpful so that users that are unfamiliar with how to debug JS issues can generate effective and useful screenshots here would be good?

Comment (BenWilson2) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:69]: (or a link to Chrome / Firefox's dev tools usage workflow)

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (BenWilson2) [inline .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:32]: Would this be better as a required checkbox?

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (BenWilson2) [inline .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:34]: Checkbox?

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:69]: +1 to adding a link to Chrome / Firefox's dev tools usage workflow.

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:73]: I'm referencing images in my fork now for testing. I'll replace them before the merge.

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:32]: Good question. To do that, we need to create a separate section. GitHub issue form doesn't support having multiple widgets in one section.

Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:32]: btw do we need this question? We normally don't care whether the reproducing code is custom or not.

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:34]: Are you suggesting something like this?

```suggestion
        - [ ] MLflow installed from source:
        - [ ] MLflow installed from binary:
```

Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:34]: Can we remove this question and ask the following instead?

```suggestion
- **MLflow version (or commit SHA if you install mlflow from source)**:
```

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (BenWilson2) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:32]: 100% agreed. Let's remove it.

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (BenWilson2) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:34]: good idea!

**Review by BenWilson2 — APPROVED** [review]
LGTM with your recommended changes!

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (dbczumar) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:32]: Agreed!

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (dbczumar) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:34]: +1

**Review by dbczumar — APPROVED** [review]
LGTM once remaining comments are addressed! Thanks @harupy !

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:95]: Yes, we can remove it! Let's remove it.

File: .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml
Comment (harupy) [reply .github/ISSUE_TEMPLATE/ui_bug_report_template.yaml:69]: Added!

**Pull request merged**
